projects/project_ideas.md

```diff
@@ -0,0 +1,59 @@
+# CS 7290: Project Ideas
+**Prof. Robert Ness**
+
+## 1. State-of-the-art Causal Inference methods
+
+You are encouraged to explore Causal Inference literature and to explore the papers that suits your research interests. For reference, take a look at the proceedings of NeurIPS and UAI for last four years. The applications of these methods do not limit to AI and computer science. They expand to Epidemiology, Social sciences, Medicines, Statistics and Political sciences. You can pick the case studies that suit your interest.
+
+**Expectations**: Comprehensive literature survey and understanding of the problem, implementation of state-of-the-art methods, comparative analysis of similar methods and suggesting possible improvements.
+
+
+## 2. Causal Inference with Probabilistc Generative Models
+
+[https://arxiv.org/abs/1805.06826]()
+
+**Introduction**:
+
+Causal inference from observational data often assumes “ignorability,” that all confounders are observed. This assumption is standard yet untestable. However, many scientific studies involve multiple causes, different variables whose effects are simultaneously of interest. *The deconfounder*, an algorithm that combines unsupervised machine learning and predictive model checking to perform causal inference in multiple-cause settings. *The deconfounder* infers a latent variable as a substitute for unobserved confounders and then uses that substitute to perform causal inference. The deconfounder provides a checkable approach to estimating closer-to-truth causal effects.
```

Comment:
This paper demonstrates a way of using a latent variable model to answer an intervention prediction question such as "How much would this film script if we produced it with DiCaprio as a star?" based on past observational data such as a database of films, who starred in them, and their box office revenue.  Another example is "Which wording would increase the click through rate of this email marketing campaign?" based on past email marketing campaign data.  Use the original data set described in the paper or find a new one.  Talk to the instructor or TA's if you have question about the appropriateness of a data set.